qucs/qucs/components/componentdialog.cpp

```diff
@@ -602,7 +602,8 @@ void ComponentDialog::slotSelectProperty(QTableWidgetItem *item)
     ButtAdd->setEnabled(true);
     ButtRem->setEnabled(true);
 
-    if (Comp->Description == "equation") {
+    // enable Up/Down buttons only for the Equation component
+    if (Comp->Model == "Eqn") {
```

Comment:
please remove that. componentdialog must be cleaned from component code. this does not improve the situation.